qml/Ping360Visualizer.qml

```diff
@@ -86,9 +86,9 @@ Item {
                         id: mouseConfidenceText
                         x: mouseReadout.width - width
                         y: mouseReadout.height*4/5
-                        text: transformValue(waterfall.mouseSampleConfidence) + "%"
-                        visible: typeof(waterfall.mouseSampleConfidence) == "number"
-                        color: confidenceToColor(waterfall.mouseSampleConfidence)
+                        text: transformValue(waterfall.mouseSampleAngle) + "º"
+                        visible: typeof(waterfall.mouseSampleAngle) == "number"
+                        color: "green"
```

Comment:
The contrast is not very good. It might be more useful to give the text a static position in the corner (not obscuring data).